cubi_isa_templates/isatab-stem_cell_core_sc/{{cookiecutter.__output_dir}}/s_{{cookiecutter.s_file_name}}.txt

```diff
@@ -1,8 +1,8 @@
-{% set sample_names = cookiecutter.sample_names.split(",") -%}
+{% set source_names = cookiecutter.source_names.split(",") -%}
 {% set cellculture_meta = cookiecutter.cellculture_meta.split(",") -%}
 {% set sample_meta = cookiecutter.sample_meta.split(",") -%}
 Source Name	Characteristics[Organism]	Term Source REF	Term Accession Number	Characteristics[Source Type]	Characteristics[Cellline]	Characteristics[Genetic modifications]	Protocol REF	Parameter Value[Passage]{%+ for meta in cellculture_meta %}	Parameter Value[{{meta}}]{% endfor %}	Protocol REF	Parameter Value[Protocol numbers]	Parameter Value[Culture type]	Parameter Value[Differentiation dates]	Parameter Value[No. of passages]	Parameter Value[Well format]	Parameter Value[Day]	Performer	Protocol REF	Parameter Value[Sample type]	Parameter Value[Collection reagent]	Performer	Date	Sample Name{%+ for meta in sample_meta %}	Characteristics[{{meta}}]{% endfor %}
-{% for sample_name in sample_names -%}
-{% set sample = sample_name|replace(" ", "_") -%}
+{% for source_name in source_names -%}
```

Comment:
This actually is the sample name and not the source name

Reply:
It's the source_name variable being re-used to define sample names. Before it was the sample_name variable being re-used to define source names.

Reply:
You're right. Reverted this change.